docs/design/Non_Admin_Controller_design.md

```diff
@@ -0,0 +1,215 @@
+# Non-Admin Backup/Restore Design
+
+## Background
+[OADP (Openshift API for Data Protection)](https://github.com/openshift/oadp-operator) Operator currently requires cluster admin access for performing Backup and Restore operations of applications deployed on the OpenShift platform.
```

Comment:
here "has" would be better than "requires"?

suppose someone tries to use OADP in non admin way (without NAC). Installs it in a non admin namespace. the non admin user can only create things in that namespace, but it can tell OADP to backup/restore anything (in any namespace) in the cluster, right?

Reply:
I think either is fine, currently we advertise OADP functionalities as cluster admin tasks so that way requires makes sense.